backend/backend/api/routes/room.py

```diff
@@ -50,7 +50,7 @@ async def enter_room(room: Annotated[Room, Depends(get_room_by_id)],
             status_code=status.HTTP_200_OK,
             response_model=list[RoomMember])
 async def get_room_players(room: Annotated[Room, Depends(get_room_by_id)]):
-    return list(room.members.values())
+    return [value for value in room.members.values()]
```

Comment:
Why? Explain the change (from list to list comp)

Reply:
I think it was becuase webSocket was not a valid type for websockets- now that websocket is pydantic type thid can be removed